Documentation/operations/upgrade.rst

```diff
@@ -327,7 +327,13 @@ Annotations:
 * Egress Gateway policies now drop matching traffic when no
   gateway nodes can be found. Previously, traffic would be allowed without
   being rerouted towards an Egress Gateway.
-
+* A bug that prevented dual-stack clusters (i.e. clusters that use both IPv4
+  and IPv6 addresses) from referencing the total address space of each IP
+  family individually (e.g. "0.0.0.0/0" and "::/0") in network policies is
+  fixed. Operators of dual-stack clusters should be aware that if they have to
+  rollback from 1.14 (because of a failed upgrade, etc) then connections to
+  endpoints external to the cluster may break, though any new connections will
+  be fine. Upgrading to 1.14 will not cause any such problems.
```

Comment:
Edits for clarity, use present tense
```suggestion
  fixed. If operators of dual-stack clusters need to
  roll back from 1.14 (for example, because of a failed upgrade), existing connections to
  endpoints external to the cluster may break, though new connections will
  work correctly. Upgrading to 1.14 does not cause this problem.
```